app/services/pii/encryptor.rb

```diff
@@ -0,0 +1,21 @@
+module Pii
```

Comment:
yes, this is not encryption per se. Just Base64 encoding. The class name is a placeholder for the concept of encrypting. Actual encryption will happen in #498